upt_macports/templates/base.Portfile

```diff
@@ -1,21 +1,27 @@
+{%- macro depends(kind, deps) -%}{% if deps %}
+    depends_{{ kind }}-append \
+                    {%- for dep in deps %}
+                port:{{ dep|reqformat }}{% if not loop.last %} \
+                    {%- endif -%}
+                    {% endfor -%}
+{% endif -%}
+{% endmacro -%}
```

Comment:
as I said earlier, the indentation of the dependencies is not correct. The code below works for me (at least after changing the ```jinja2.Environment``` arguments to their defaults):
```
{%- macro depends(kind, deps) -%}
{%- if deps %}
    depends_{{ kind }}-append \
  {%- for dep in deps %}
                    port:{{ dep|reqformat }} {%- if not loop.last %} \ {%- endif -%}
  {%- endfor -%}
{%- endif -%}
{%- endmacro -%}
```